.drone.star

```diff
@@ -0,0 +1,136 @@
+PLATFORMS = [
+    {"os": "linux", "arch": "amd64"},
+    {"os": "linux", "arch": "arm64"},
+    {"os": "darwin", "arch": "amd64"},
```

Comment:
Anything blocking `{darwin: "arm64"}` on the other Mac Mini?

Reply:
nothing except that i had not yet provided drone-runner with updated config.

Reply:
darwin-arm64 is not supported by drone runners :(